docs/reference/errors-and-warnings/NU3037.md

```diff
@@ -18,6 +18,7 @@ A NuGet package signature has expired.
 A package signature shares the same validity period as the certificate used to generate the signature. A package signature is invalid outside of that validity period.
 To ensure long-term validity --- even beyond the signing certificate’s validity period --- a package signature should be timestamped with a trusted timestamp. Trusted timestamps must be added while a package signature is still valid and not expired.
 
+On Windows only, NU3037 may occur the first time a root certificate is observed and with the message "The repository primary signature validity period has expired."  If the issue is resolved with retries, [there is an option which may help](../Signed-Package-Verification-Options.md#retry-untrusted-root-failures).
```

Comment:
```suggestion
On Windows only, NU3037 may occur the first time a root certificate is observed and with the message, "The repository primary signature validity period has expired."  If the issue is resolved with retries, [there is an option which may help](../Signed-Package-Verification-Options.md#retry-untrusted-root-failures).
```

Reply:
I researched this and I _think_ it's OK as is.  Search:  ["when not to use comma before a quote"](https://www.bing.com/search?q=when+to+not+use+a+comma+before+a+quote).  It seems like the quote acts like a grammatical part of the sentence.  Another [similar example](https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/quotation-mark-rules):  `Does he always say “I’m going to be late to class today”?`